scene/3d/arvr_nodes.cpp

```diff
@@ -69,6 +69,13 @@ String ARVRCamera::get_configuration_warning() const {
 		warning += TTR("ARVRCamera must have an ARVROrigin node as its parent.");
 	};
 
+	if (is_physics_interpolated()) {
+		if (warning != String()) {
+			warning += "\n\n";
+		}
+		warning += TTR("Physics interpolation in ARVRCamera isn't needed and should be off to avoid jitter.");
```

Comment:
It may be worth changing the wording here slightly, as users often interpret this kind of thing as saying project setting physics interpolation (global) should be off.

Maybe something like "ARVRCamera should have physics_interpolation_mode set to OFF in order to avoid jitter.".